### PR TITLE
Made it possible to pass multiple files to pandoc - fixes #248

### LINKED ISSUE #248: Possible to pass multiple files to `convert_file`
At the command line this is okay:

```
pandoc _posts/*.md \
  --from markdown \
  -o books/cookbook.epub
```

but it looks like if I do the same with `pypandoc.convert_file('_posts/*.md', 'epub', outputfile="books/test.epub")` I get this error:

```
Traceback (most recent call last):
  File "book_creator.py", line 26, in <module>
    pypandoc.convert_file('_posts/2022-01-12-vegan-shakshuka.md _posts/2022-01-12-vegan-dahl.md', 'epub', outputfile="books/test.epub")
  File "/home/simon/src/symbiosis-cookbook/.venv/lib/python3.8/site-packages/pypandoc/__init__.py", line 148, in convert_file
    raise RuntimeError("source_file is not a valid path")
RuntimeError: source_file is not a valid path
```

So I tried passing multiple files to the converter, listed individually:

```
pypandoc.convert_file('_posts/2022-01-12-vegan-shakshuka.md _posts/2022-01-12-vegan-dahl.md', 'epub', outputfile="books/test.epub")
```

which gives the same error.

Is there a way to pass multiple files to pandoc like on the command line? (I'm basically just using python to sort the markdown files by frontmatter category).

### PR DESCRIPTION
**Pull request by NicklasTegner**
This pr makes it possible to pass multiple files to pandoc in various ways.

Examples (from the updated readme)

```python
import pypandoc

# convert all markdown files in a chapters/ subdirectory.
pypandoc.convert_file('chapters/*.md', 'docx', outputfile="somefile.docx")

# convert all markdown files in the book1 and book2 directories.
pypandoc.convert_file(['book1/*.md', 'book2/*.md'], 'docx', outputfile="somefile.docx")

# convert the front from another drive, and all markdown files in the chapter directory.
pypandoc.convert_file(['D:/book_front.md', 'book2/*.md'], 'docx', outputfile="somefile.docx")
```

This fixes #248 

simonv3

### CODE REVIEW

**Comment by simonv3** [comment]
This looks like it should do what I want it to, thanks @NicklasTegner

File: tests.py [hunk 1]
```diff
@@ -193,6 +193,25 @@ def test_basic_conversion_from_file(self):
...
+        with closed_tempfile('.md', text='some title') as file_name1:
+            with closed_tempfile('.md', text='some title') as file_name2:
+                expected = '<p>some title</p>\n<p>some title</p>'
+                received = pypandoc.convert_file([file_name1,file_name2], 'html')
+                self.assertEqualExceptForNewlineEnd(expected, received)
+
+    def test_basic_conversion_from_file_pattern(self):
+        received = pypandoc.convert_file("./*.md", 'html')
+        received = received.lower()
+        assert "making a release" in received
```
Comment (hugobuddel) [inline tests.py:206]: This `making a release`-snipped originates from [release.md](https://github.com/JessicaTegner/pypandoc/blob/master/release.md). However, that file is not shipped with the release (it is not mentioned in the manifest). Therefore the tests fail in the released packages. Perhaps `release.md` should be added to `MANIFEST.in`.

Comment (JessicaTegner) [reply tests.py:206]: hmmm. Just tried building locally (with poetry build) and as when we push a release we make both a wheel and a tarball, and it's different which *.md files are actually included in the specific package.

So I propose to rewrite the tast case instead, using .md files from the "test-data" directory.

What are your thoughts on that?

Comment (hugobuddel) [reply tests.py:206]: Rewriting the test is fine with me too.

Choosing a different set of md files for that test was my first thought as well; then I saw that [filter_test.md](https://github.com/JessicaTegner/pypandoc/blob/master/filter_test.md) is also included in the manifest for testing purposes, so I decided to follow that example. But maybe that was not such a good example to start from.

**Pull request merged**